octopoes/octopoes/core/service.py

```diff
@@ -45,6 +46,15 @@
 
 logger = getLogger(__name__)
 settings = Settings()
+BIT_CACHE: dict[int, list[OOI]] = {}
```

Comment:
We should use an existing Caching implementation, as this cache will keep on growing for ever. Handling Cache eviction could be done based on the Primary-key of each OOI (input or param) and its current and old/stale content hash, however that seems like a lot of work for this 'easy' fix. An LRU might be more useful for now.

Reply:
Addressed in 3805e96b96bd9cc4811cb4744ed02b0c5c27e4d0

Reply:
Addressed in a9ff441948996c1e6d66a3464c5d5e2f95088da9.